final/client/src/pages/login.js

```diff
@@ -11,26 +15,26 @@ export const LOGIN_USER = gql`
 `;
 
 export default function Login() {
+  const client = useApolloClient();
+  const [login, { loading, error }] = useMutation(
+    LOGIN_USER,
+    {
+      onCompleted({ login }) {
+        localStorage.setItem('token', login);
+        client.writeData({ data: { isLoggedIn: true } });
+      }
+    }
+  );
+
   return (
     <ApolloConsumer>
```

Comment:
Can we get rid of the `ApolloConsumer` component here now that we're using the `useApolloClient` hook?

Reply:
Oops - definitely. Great catch!